public/css/style.css

```diff
@@ -1,5 +1,9 @@
 /* Great source: https://css-tricks.com/hexagons-and-beyond-flexible-responsive-grid-patterns-sans-media-queries/ */
 
+form {
```

Comment:
Since your css is global for the moment, I think you have to be more precise in the selector